### PR TITLE
Fixed Sonar bugs

### PR DESCRIPTION
**Pull request by aihamh**

### CODE REVIEW

File: resident/resident-service/src/main/java/io/mosip/resident/constant/PacketStatus.java [hunk 1]
```diff
@@ -37,7 +37,11 @@ public static Optional<String> getStatusCode(String statusCode, Environment env)
 	}
 
 	private static List<String> getStatusCodeList(PacketStatus packetStatus, Environment env) {
-		return List.of(env.getProperty(packetStatus.statusCodePropertyName).split(","));
+		List<String> statusCode = null;
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/constant/PacketStatus.java]: Initialize it with empty list. (List.of())

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utility.java [hunk 2]
```diff
@@ -172,7 +172,9 @@ public Map<String, String> getAmrAcrMapping() throws ResidentServiceCheckedExcep
 				String.class);
 		Map<String, Object> amrAcrMap = null;
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/util/Utility.java]: Initialize with empty map (Map.of())

File: resident/resident-service/src/main/java/io/mosip/resident/constant/TransactionStage.java [hunk 3]
```diff
@@ -33,7 +33,11 @@ public static Optional<String> getTypeCode(String transactionTypeCode, Environme
 	}
 
 	private static List<String> getTransactionTypeCodeList(TransactionStage transactionStage, Environment env) {
-		return List.of(env.getProperty(transactionStage.transactionTypeCodePropertyName).split(","));
+		List<String> transactionTypeCode = null;
```
Comment (loganathan-sekaran) [inline resident/resident-service/src/main/java/io/mosip/resident/constant/TransactionStage.java]: Initialize it with empty list. (List.of())

**Review by loganathan-sekaran — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/main/java/io/mosip/resident/constant/PacketStatus.java (hunk 1 above)
Comment (aihamh) [reply resident/resident-service/src/main/java/io/mosip/resident/constant/PacketStatus.java]: Updated as per review comment

File: resident/resident-service/src/main/java/io/mosip/resident/util/Utility.java (hunk 2 above)
Comment (aihamh) [reply resident/resident-service/src/main/java/io/mosip/resident/util/Utility.java]: Updated as per review comment

File: resident/resident-service/src/main/java/io/mosip/resident/constant/TransactionStage.java (hunk 3 above)
Comment (aihamh) [reply resident/resident-service/src/main/java/io/mosip/resident/constant/TransactionStage.java]: Updated as per review comment

**Review by loganathan-sekaran — APPROVED** [review]

**Pull request merged**